src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -276,6 +280,14 @@ void CLibLinear::solve_l2r_l1l2_svc(
 	// default solver_type: L2R_L2LOSS_SVC_DUAL
 	double diag[3] = {0.5/Cn, 0, 0.5/Cp};
 	double upper_bound[3] = {CMath::INFTY, 0, CMath::INFTY};
+
+	SGVector<float64_t> linear_term;
+	if (linear_term_inited())
+	{
+		SGVector<float64_t> linear_term = get_linear_term();
```

Comment:
mmm, but it's named `m_ linear_term ` as class variable  in https://github.com/shogun-toolbox/shogun/blob/develop/src/shogun/classifier/svm/LibLinear.h#L215 :)